include/hc_rt_debug.h

```diff
@@ -43,25 +43,21 @@ static std::vector<std::string> g_DbStr = {"api", "cmd", "wait", "aql", "queue",
 // DBOUT(" Something happened" << myId() << " i= " << i << "\n");
 #define COMPILE_HCC_DB 1
 
-#define DBFLAG(db_flag) (HCC_DB & (1<<(db_flag)))
+#define DBFLAG(db_flag) (COMPILE_HCC_DB && (HCC_DB & (1<<(db_flag))))
 
-#define DBSTREAM std::cerr
+#define DBSTREAM  std::cerr
+#define DBWSTREAM std::wcerr
 
 // Use str::stream so output is atomic wrt other threads:
 #define DBOUT(db_flag, msg) \
-if (COMPILE_HCC_DB && (HCC_DB & (1<<(db_flag)))) { \
+if (DBFLAG(db_flag)) { \
```

Comment:
Would it be useful to add DBWFLAG which can replace the debug outputs that look like:
```
        if (DBFLAG(DB_INIT)) {
          DBWSTREAM << L"Path: " << path << L"\n";
          DBWSTREAM << L"Description: " << description << L"\n";
        }
```
with `DBWOUTL(DB_INIT,  L"Path: " << path << L);` and `DBWOUTL(DB_INIT,  L"Description: " << description << L);`

Reply:
Going through the runtime code, I could only find once instance where the extra macro would help so I would not add DBWOUT at this point.